### ISSUE TITLE
no search?

### ISSUE THREAD

**Issue by kai-uwe-rommel** [TandoorRecipes/recipes#2733]
### Tandoor Version

1.1.4

### Setup

Synology

### Reverse Proxy

No reverse proxy

### Bug description

I did fresh setup, a few recipes were now entered.
When I search for a word from a text in a recipe, nothing is found.
What can one search for? Is that not a full text search?

**Comment by kai-uwe-rommel** [comment]
Ok, looks like it just takes a while after which search starts working.

**Comment by kai-uwe-rommel** [comment]
After a couple of days I have to say that search does not really work reliably.
Consider this:
![grafik](https://github.com/TandoorRecipes/recipes/assets/26381804/e9f734b5-1acf-4b93-921c-1f048845dff4)
When I now search for it:
![grafik](https://github.com/TandoorRecipes/recipes/assets/26381804/8fe494f8-f1ef-4922-93a6-b464101277bf)
What can I do to fix this?

**Comment by kai-uwe-rommel** [comment]
Search works, when I switch from Fuzzy to Precise in the Search Settings.

**Comment by vabene1111** [comment]
Yes, there are a few issues with the search sadly, I had your exact problem many times before. usually the fuzzy preset should solve it but i also plan on doing some significant changes to the search in the future where this will be addressed as well.

Closing this issue as the underlying reason is much more complex and I will consider it in a seperate project.

**Comment by kai-uwe-rommel** [comment]
But for me it (currently ...) is the other way around, e.g. Precise works. Fuzzy does not work.
